tools/methylkit/methylkit.xml

```diff
@@ -0,0 +1,519 @@
+          <expand macro="differential_methylation" />
+          <expand macro="clustering" />
+        </when>
+        <when value="differential_methylation">
+          <expand macro="differential_methylation" />
+        </when>
+        <when value="clustering">
+          <expand macro="clustering" />
+        </when>
+        <when value="segmentation" />
+    </conditional>
+  </inputs>
+  <outputs>
+      <data name="output_statistics" format="pdf"
+      from_work_dir="output_statistics.pdf"
+      label="${tool.name} on ${on_string}: CpG statistics">
+      </data>
+
+      <data name="output_correlation" format="pdf"
+      from_work_dir="output_correlation.pdf"
+      label="${tool.name} on ${on_string}: correlation between samples">
+      </data>
+
+      <data name="output_myDiff" format="bedgraph"
+      from_work_dir="output_myDiff.bed"
+      label="${tool.name} on ${on_string}: differential methylation">
+      <filter>input_type['choice'] in ['all', 'differential_methylation']</filter>
+      </data>
+
+      <data name="output_MethylDiff" format="bedgraph"
+      from_work_dir="output_MethylDiff.bed"
+      label="${tool.name} on ${on_string}: differential methylation - subset">
+      <filter>input_type['choice'] in ['all', 'differential_methylation']</filter>
+      </data>
+
+      <data name="output_MethPerChr" format="csv"
+      from_work_dir="output_MethPerChr.csv"
+      label="${tool.name} on ${on_string}: number of hyper/hypo sites">
+      <filter>input_type['choice'] in ['all', 'differential_methylation']</filter>
+      </data>
+
+      <data name="output_clustering" format="pdf"
+      from_work_dir="output_clustering.pdf"
+      label="${tool.name} on ${on_string}: hierarchical clustering">
+      <filter>input_type['choice'] in ['all', 'clustering']</filter>
+      </data>
+
+      <data name="output_PCA" format="pdf"
+      from_work_dir="output_PCA.pdf"
+      label="${tool.name} on ${on_string}: PCA">
+      <filter>input_type['choice'] in ['all', 'clustering']</filter>
+      </data>
+
+      <data name="output_seg_raw" format="bed"
+      from_work_dir="output_seg_raw.bed"
+      label="${tool.name} on ${on_string}: methylation segment">
+      <filter>input_type['choice'] in ['all', 'segmentation']</filter>
+      </data>
+
+      <data name="output_seg_diff" format="bed"
+      from_work_dir="output_seg_diff.bed"
+      label="${tool.name} on ${on_string}: differential methylation segment">
+      <filter>input_type['choice'] in ['all', 'segmentation']</filter>
+      </data>
+  </outputs>
+  <tests>
+    <test>
+        <repeat name="test_series">
+            <param name="input" value="input_test1.myCpG.txt" ftype="txt" />
+        </repeat>
+        <repeat name="test_series">
+            <param name="input" value="input_test2.myCpG.txt" ftype="txt" />
+        </repeat>
+        <repeat name="control_series">
+            <param name="input" value="input_control1.myCpG.txt" ftype="txt" />
```

Comment:
No only output, just wanted to point it out :) Sorry for the confusion. But here we should label it as tabular.